README.md

```diff
@@ -1,389 +1,59 @@
-
-- #### [Delete event](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/delete_event.bal)
-
-    This sample shows how to delete an event in an authorized user's calendar. The calendar and event ids are required to do this operation. This operation returns an error `true` if unsuccessful. 
-
-- #### [Watch event changes](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/watch_event.bal)
-
-    This sample shows how to watch for changes to events in an authorized user's calendar. It is a subscription to receive push notification from Google on events changes.  The calendar id and callback url are required to do this operation. Channel live time can be provided via an optional parameter. By default it is 604800 seconds. This operation returns  `WatchResponse` if successful. Else returns `error`.
-
-- #### [Stop channel](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/stop_channel.bal)
-
-    This sample shows how to stop an existing subscription. The channel id and resource is are required to do this operation. This operation returns an error `true` if unsuccessful.
-
-## Listener
-
-- #### [Trigger for new event](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/trigger_create_event.bal)
-
-    This sample shows how to create a trigger on new event. When a new event is occurred, that event details can be captured in this listener.
-
-- #### [Trigger for updated event](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/trigger_update_event.bal)
-
-    This sample shows how to create a trigger on an event update. When a new event is updated, that event details can be captured in this listener.
-
-- #### [Trigger for deleted event](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/trigger_delete_event.bal)
-
-    This sample shows how to create a trigger on cancelled event. When a new event is cancelled, that event details can be captured in this listener.
+[Google Calendar](https://developers.google.com/calendar) is a time-management and scheduling calendar service developed by Google. It lets users to organize their schedule and share events with others.
+
+This connector provides the capability to programmatically manage events and calendar and listen the events occurred in the calendar. 
+For more information about configuration and operations, go to the module.
+- [ballerinax/googleapis.calendar](https://docs.central.ballerina.io/ballerinax/googleapis.calendar/0.1.9)
+- [ballerinax/googleapis.calendar.listener](https://docs.central.ballerina.io/ballerinax/googleapis.calendar.listener/0.1.9)
```

Comment:
The latest version of calendar in central is 0.1.8
But in this beta2 release, we are planning to change version to 1.0.0

Reply:
Fixed. Thanks